server/export.js

```diff
@@ -27,9 +27,12 @@ module.exports = function(fs, db) {
             });
         }) // run the command to output the table to a file
         // replace 'annotation' with a select query to get certain rows instead of the whole thing
-        .then(db.none(`copy annotation to '/tmp/csv/export.csv' DELIMITER ',' CSV HEADER`))
+        .then(db.none(`COPY annotation TO '/tmp/csv/export.csv' DELIMITER ',' CSV HEADER`))
         .then(function() {
             // download file to client
+            if (!fs.existsSync('/tmp/csv/export.csv')){
+                fs.writeFile('/tmp/csv/export.csv', 'content')
```

Comment:
these should be deleted, synchronous IO is bad for performance and the check is unneccessary because I catch the error in res.download if the fie doesn't exist for whatever reason